plugins/modules/win_dns_record.py

```diff
@@ -19,6 +19,20 @@
     - The name of the record.
     required: yes
     type: str
+  port:
+    description:
+    - The port number of the record.
+    - Optional for C(type=SRV).
+    - Supported only for C(type=SRV).
+    default: ""
```

Comment:
Default values for int types should be an actual number not an empty string.

Reply:
I thought for `-Port` value 0 is not making any sense because Port 0 is invalid.

Reply:
That's why it doesn't make sense having that as the default value, it shouldn't have any defaults and just be "required" if `type: SRV`. That can't be documented with `required = $true` but rather in the `description` for the option and using that `required_if` statement I had written.

Reply:
Okay Understood